README.md

```diff
@@ -723,6 +723,38 @@ The Bundle's `link` array will include a link with relation=`next` if more resul
 Otherwise the array will contain a `link` with relation=`sync` that can be used to check for new results in
 one minute.
 
+### Cancellation/Amending/Restoring Physical Measurements
+### `PATCH /Participant/:pid/PhysicalMeasurements/:id`
+
+Payload for cancelling/restoring/amending a physical measurement looks like this:
+`{
```

Comment:
triple ticks would make this a mulitline code block instead of an inline code span

Reply:
done